plugins/uefi/fu-uefi-device.c

```diff
@@ -356,10 +356,10 @@ fu_uefi_device_fixup_firmware (FuDevice *device, GBytes *fw, GError **error)
 	/* ESRT header matches payload */
 	if (g_strcmp0 (fu_uefi_device_get_guid (self), guid_new) == 0) {
 		g_debug ("ESRT matches payload GUID");
-		return g_bytes_new_from_bytes (fw, 0, fw_length);
+		return g_bytes_ref (fw);
 	/* Type that doesn't require a header */
 	} else if (!self->requires_header) {
-		return g_bytes_new_from_bytes (fw, 0, fw_length);
+		return g_bytes_ref (fw);
```

Comment:
shouldn't this be a separate commit?

Reply:
I don't think so; I didn't convert g_bytes_new_from_bytes() here as it wasn't actually required.

Reply:
Right, my point was this change could have been done separately and didn't need to come in the exact same commit.  Doesn't really matter for anything but backports and bisects.